libenkf/src/model_config.c

```diff
@@ -807,10 +769,6 @@ static void model_config_init_user_config(config_parser_type * config ) {
   /*****************************************************************/
   /* Required keywords from the ordinary model_config file */
 
-  item = config_add_schema_item(config, CASE_TABLE_KEY, false);
```

Comment:
Sorry for screaming about the documentation all the time, but...

Reply:
The `CASE_TABLE` keyword has been removed from the documentation.